mmdet/models/dense_heads/ga_rpn_head.py

```diff
@@ -69,6 +69,15 @@ def _get_bboxes_single(self,
                            cfg,
                            rescale=False):
         cfg = self.test_cfg if cfg is None else cfg
+
+        # refactor the nms cfg
+        # this is used for avoid breaking change
+        if 'nms' not in cfg:
+            cfg.nms = dict(type='nms', iou_threshold=cfg.nms_thr)
+            cfg.max_per_img = cfg.max_num
+
+        assert cfg.nms.get('type', 'nms') == 'nms', 'GARPNHead only support ' \
```

Comment:
I think it’s strange, whether `assert cfg.nms.type == 'nms'` would be better.

Reply:
We support to not specify the `type` in `cfg.nms` and use naive nms in this case.